### DISCUSSION TITLE
[Feature] set album description

### DISCUSSION THREAD

**Discussion by mspoorendonk** [immich-app/immich#13024] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### Context and dependency

My library is organised in folders that represent albums. The folders contain album descriptions in text files. When loading an external library, I'd like immich to create albums on the fly. See https://github.com/immich-app/immich/discussions/4279

### The feature

When that import happens, I'd like Immich to read the descriptions for those albums from the text files in those folders.

Currently these are various legacy text files such as picasa.ini. I think it's perfectly fine however to have the user first convert any legacy text files into some kind of standard format such as XMP, json or yaml. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
This is multiple feature requests in one with some duplicates, both of which the template asks you to avoid.

**Reply by mspoorendonk** [reply]
You are perfectly right, I rewrote the text to reference the existing feature request, and removed the included sharing feature request.
